App/Logic/ExposureDetectionLogic.swift

```diff
@@ -251,6 +237,7 @@ extension Logic.ExposureDetection {
           continue
         }
         for (index, countryOfInterest) in state.exposureDetection.countriesOfInterest.enumerated() {
+          //swiftlint:disable for_where
```

Comment:
<!--
  0 failure: 
  1 warning:  swiftlint (0.39.2...
  
  
  DangerID: danger-id-Danger;
  File: App/Logic/ExposureDetectionLogic.swift;
  Line: 240;
-->

- :warning: swiftlint (0.39.2): `where` clauses are preferred over a single `if` inside a `for`. (for_where)